odoo/netsvc.py

```diff
@@ -85,13 +85,19 @@ class PerfFilter(logging.Filter):
     def format_perf(self, query_count, query_time, remaining_time):
         return ("%d" % query_count, "%.3f" % query_time, "%.3f" % remaining_time)
 
+    def format_cursor_mode(self, cursor_mode):
+        return cursor_mode or 'na'
+
     def filter(self, record):
         if hasattr(threading.current_thread(), "query_count"):
             query_count = threading.current_thread().query_count
             query_time = threading.current_thread().query_time
             perf_t0 = threading.current_thread().perf_t0
             remaining_time = time.time() - perf_t0 - query_time
             record.perf_info = '%s %s %s' % self.format_perf(query_count, query_time, remaining_time)
+            if tools.config['db_replica_host'] is not False:
+                cursor_mode = threading.current_thread().cursor_mode
+                record.perf_info = f'{record.perf_info} {self.format_cursor_mode(cursor_mode)}'
```

Comment:
I don't think it is a good idea to have variation in the logs... I mean... we do print query_count and query_time even for auth='none' requests

Reply:
on the other hand changing the log line for everybody will have important consequences, I can imagine scripts running in the wild that expect the find 3 digits at the end

Reply:
and again, if we are to set the cursor session readonly even on the read/write pool, logging a red "ro->rw" also in development (where we definitely are *not* going to setup a replica host) is a good idea

Reply:
I can feel that this oine is going to get political... may the force be with you :pray:

Reply:
It is actually column based so it remains the same column at the end

`2024-01-03 21:42:48,556 15 INFO master-all werkzeug: 123.123.123.123 - - [03/Jan/2024 21:42:48] "GET / HTTP/1.0" 200 - 174 0.106 0.732`
This is the purpose of the - in all case, to have a fixed number of columns ;) here I'm just adding one.  
Making it conditional if we have or don't have the replica remains a good question.